lib/src/http/w_request.dart

```diff
@@ -56,12 +54,18 @@ abstract class WRequest implements FluriMixin {
   /// [WProgress] stream for this HTTP request's download.
   Stream<WProgress> get downloadProgress;
 
+  /// Map of HTTP request headers
+  CaseInsensitiveMap get headers;
```

Comment:
This public type should still be `Map<String, String>`. It will still work since `CaseInsenstiveMap<String> implements Map<String, String>`.